### PR TITLE
[zh] Sync contribution changes (1)

### PR DESCRIPTION
**Pull request by tengqm**
There have been some significant changes to the contribution guide. This PR is one of the series that sync the changes to Chinese localization.

### CODE REVIEW

File: content/zh/docs/contribute/participate/roles-and-responsibilties.md [hunk 1]
```diff
@@ -0,0 +1,409 @@
...
+   通过 [Kubernetes Slack 上的 #sig-docs 频道](https://kubernetes.slack.com) 或者
+   [SIG Docs 邮件列表](https://groups.google.com/forum/#!forum/kubernetes-sig-docs)
+   来寻找为你担保的人。
+
+   {{< note >}}
+   不要单独发送邮件给某个 SIG Docs 成员或在 Slack 中与其私聊。
+   在提交申请之前，一定要先确定担保人。
+   {{< /note >}}
+
+2. 在 [`kubernetes/org`](https://github.com/kubernetes/org/)` 仓库
```
Comment (SataQiu) [inline content/zh/docs/contribute/participate/roles-and-responsibilties.md:147]:
```suggestion
2. 在 [`kubernetes/org`](https://github.com/kubernetes/org/) 仓库
```

File: content/zh/docs/contribute/participate/roles-and-responsibilties.md [hunk 2]
```diff
@@ -0,0 +1,409 @@
...
+
+When you meet the
+[requirements](https://github.com/kubernetes/community/blob/master/community-membership.md#reviewer), you can become a SIG Docs reviewer. Reviewers in other SIGs must apply separately for reviewer status in SIG Docs.
+
+To apply:
+-->
+### 成为评阅人   {#becoming-a-reviewer}
+
+当你满足[相关条件](https://github.com/kubernetes/community/blob/master/community-membership.md#reviewer)时，
+你可以称为一个 SIG Docs 评阅人。
```
Comment (SataQiu) [inline content/zh/docs/contribute/participate/roles-and-responsibilties.md:264]:
```suggestion
你可以成为一个 SIG Docs 评阅人。
```

File: content/zh/docs/contribute/participate/roles-and-responsibilties.md [hunk 3]
```diff
@@ -0,0 +1,409 @@
...
+
+2. Assign the PR to one or more SIG-Docs approvers (user names listed under `sig-docs-{language}-owners`).
+
+If approved, a SIG Docs lead adds you to the appropriate GitHub team. Once added, [K8s-ci-robot](https://github.com/kubernetes/test-infra/tree/master/prow#bots-home) assigns and suggests you as a reviewer on new pull requests.
+-->
+1. 发起 PR，将你的 GitHub 用户名添加到 `kubernetes/website` 仓库中
+   [OWNERS_ALIASES](https://github.com/kubernetes/website/blob/master/OWNERS)
+   文件的特定节。
+
+  {{ note }}
```
Comment (SataQiu) [inline content/zh/docs/contribute/participate/roles-and-responsibilties.md:286]: 渲染失败，https://deploy-preview-22241--kubernetes-io-master-staging.netlify.app/zh/docs/contribute/participate/roles-and-responsibilties/#becoming-a-reviewer

File: content/zh/docs/contribute/participate/roles-and-responsibilties.md [hunk 4]
```diff
@@ -0,0 +1,409 @@
...
+
+If approved, a SIG Docs lead adds you to the appropriate GitHub team. Once added, [K8s-ci-robot](https://github.com/kubernetes/test-infra/tree/master/prow#bots-home) assigns and suggests you as a reviewer on new pull requests.
+-->
+申请流程如下：
+
+1. 发起一个 PR，将自己添加到 `kubernetes/website` 仓库中
+   [OWNERS_ALIASES](https://github.com/kubernetes/website/blob/master/OWNERS)
+   文件的对应节区。
+
+    {{ note }}
```
Comment (SataQiu) [inline content/zh/docs/contribute/participate/roles-and-responsibilties.md:400]: 渲染失败，https://deploy-preview-22241--kubernetes-io-master-staging.netlify.app/zh/docs/contribute/participate/roles-and-responsibilties/#becoming-an-approver

File: content/zh/docs/contribute/localization.md [hunk 5]
```diff
@@ -482,79 +501,81 @@ To collaborate on a development branch:
 
 4. Periodically, an approver merges the development branch to its source branch by opening and approving a new pull request. Be sure to squash the commits before approving the pull request. 
 -->
-2. 个人贡献者基于开发分支新建特性分支。
+1. 个人贡献者基于开发分支创建新的特性分支
```
[the remark this answers is not in the document]
Comment (tengqm) [reply content/zh/docs/contribute/localization.md:504]: ok, will use explicit numbers then

File: content/zh/docs/contribute/participate/roles-and-responsibilties.md (hunk 3 above)
Comment (tengqm) [reply content/zh/docs/contribute/participate/roles-and-responsibilties.md:286]: 经验：缩进三格以上，四格靠谱

**Pull request merged**